03_Modules/Tenant/src/module/api.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2023 S44, LLC
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { AbstractModuleApi, CallAction, Namespace } from '@citrineos/base';
+import { TenantModule } from './module';
+import { ITenantModuleApi } from './interface';
+import { FastifyInstance } from 'fastify';
+import { ILogObj, Logger } from 'tslog';
+
+/**
+ * Server API for the Tenant module.
+ */
+export class TenantModuleApi
+  extends AbstractModuleApi<TenantModule>
+  implements ITenantModuleApi
+{
```

Comment:
it seems the github actions linter complains here, but i ran the `npm run lint-fix` command locally and arrived at this formatting 🤔 should there be another local command i run instead?